### DISCUSSION TITLE
Using DDSRecord and Replay on Discovery Servers

### DISCUSSION THREAD

**Discussion by Ryan-Red** [eProsima/DDS-Record-Replay#142] [Q&A]
Hello, I just wanted to ask if discovery servers have been implemented yet, as our network architecture utilizes a few servers and we'd like to record on at least one server. We have tried using 
```export ROS_SUPER_CLIENT=TRUE``` and ```export ROS_DISCOVERY_SERVER="IP:PORT"```, just like with fast-dds-spy, but it doesn't work here.

**Comment by juanlofer-eprosima** [comment]
Hi @Ryan-Red , this feature is currently not supported and its development is not present in the product roadmap. However, please feel free to contact `support@eProsima.com` so we can further analyze your use case and propose a solution.
